simulation.py

```diff
@@ -18,7 +19,7 @@
 
 def run_simulation(request_log_file, enable_concurrency = True, request_map = None):
   # replaying request log
-  start_time, end_time, request_map = replay_log.simulate_requests(request_log_file, enable_concurrency, request_map)
+  start_time, end_time, request_map,uuid_to_server = replay_log.simulate_requests(request_log_file, enable_concurrency, request_map)
```

Comment:
hmm, this was a walk-around before server can return file list. let me discard this.